docs/src/componentDocs/ScoreCard/examples/ScoreCardWithScoreBadge.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Box } from '@mui/material';
+import { CodeBlock, CodeBlockActionButtonRow } from '../../../shared';
+import * as colors from '@brightlayer-ui/colors';
+import { ScoreCardWithScoreBadgeExample } from './ScoreCardWithScoreBadgeExample';
+
+const codeSnippet = `<ScoreCard
+    sx={{ width: 350 }}
+    headerTitle={'Substation 3'}
+    headerSubtitle={'High Humidity Alarm'}
+    headerInfo={'4 Devices'}
+    actionRow={
+        <List sx={{ p: 0 }}>
+            <InfoListItem dense chevron title="View Location" hidePadding />
+        </List>
+    }
+    badge={
+        <HeroBanner>
+            <Hero
+                icon={<GradeA fontSize={'inherit'} />}
+                label={'Grade'}
+                iconSize={72}
+                iconBackgroundColor={colors.white[50]}
```

Comment:
```suggestion
                iconBackgroundColor={'background.paper'}
```

Reply:
(I don't know if our component can actually take these MUI v5 inputs... please ignore this comment if things breaks